pyproject.toml

```diff
@@ -47,7 +48,8 @@ dependencies = [
     "aiohttp",
     "importlib_metadata >=4.8.3 ; python_version<\"3.10\"",
     "jupyter-server >=1.0",
-    "simpervisor >=0.4",
+    "simpervisor >=0.4 ; platform_system!=\"Windows\"",
+    "simpervisor >=1.0 ; platform_system==\"Windows\"",
```

Comment:
Simpervisor 1.0.0 is not a breaking change for this package that already requires py38, so if we want we can just declare >=1.0 no matter what platform and that should be fine I figure.

Reply:
This warms my downstream heart.